### PR TITLE
bug 1683027 clarifying the initialization phase

### PR DESCRIPTION
**Pull request by kalexand-rh**
From https://bugzilla.redhat.com/show_bug.cgi?id=1683027

@qinpingli, is this clearer?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL.

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
+Before you deploy a {product-title} cluster, you provide parameter values to
 describe your Amazon Web Services (AWS) account and optionally customize your
-cluster's platform.  During the initialization phase of installation, you
-provide values for the required parameters through the command line. If you
-customize your cluster, you can modify the `install-config.yaml` file to
-provide more details about the platform.
+cluster's platform. When you create the `install-config.yaml` installation
+configuration file, you provide values for the required parameters through the
+command line. If you customize your cluster, you can modify the
+`install-config.yaml` file to provide more details about the platform.
```
Comment (mburke5678) [inline modules/installation-configuration-parameters.adoc]: @kalexand-rh _through the command line_ seems a bit odd here (redundant?). Maybe something like:

`Before you deploy a {product-title} cluster, you provide parameter values to describe your Amazon Web Services (AWS) account and optionally customize your cluster's platform by modifying the `install-config.yaml` installation configuration file.`

**Comment by qinpingli** [comment]
LGTM

File: modules/installation-configuration-parameters.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/installation-configuration-parameters.adoc]: @mburke5678, you have to provide some of the values through the command line to generate the file. After you generate the file, you make more changes to it. On another bug, QE's requested to make it clear that you have to start from the generated file, not copy-paste the YAML from somewhere. You're right that it sounds a bit odd, but your version makes it sound like you can make all the changes directly in the YAML file.

**Pull request merged**